plugins/mutagent/skills/mutagent-cli/references/workflows/eval-creation.md

```
@@ -0,0 +1,207 @@
+- Output Standards format for OUTPUT-scoped criteria
+- Format requirements (`name`, `description`, `evaluationParameter`)
+- Anti-patterns to avoid
+
+The CLI's `--guided` directive also contains a self-sufficient inline version of these
+rules in `_directive.instruction` -- safe to execute even without the Skill loaded.
+
+---
+
+## Workflow steps
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)